### ISSUE TITLE
Does this app support adding optimizer arguments?

### ISSUE THREAD

**Issue by Astral793** [gazingstars123/Anima-Standalone-Trainer#54]
Ppl usually recommend adding the following optimizer arguments when using Prodigy:

betas=(0.9, 0.99) safeguard_warmup=True use_bias_correction=True

I ended up editing the config.toml and _merged_config.toml in the jobs folder to look like this:

optimizer_type = "Prodigy"
optimizer_args = [
"weight_decay=0.01",
"decouple=True",
"betas=(0.9,0.99)",
"safeguard_warmup=True",
"use_bias_correction=True"
]
lr_scheduler = "cosine"

I do see the arguments in my training log, but I'm not sure if they are actually doing anything.

I also see that the makers of Prodigy recommend the following settings for the scheduler:

"As a rule of thumb, we recommend either using no scheduler or using cosine annealing with the method:

n_epoch is the total number of epochs to train the network
scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(opt, T_max=total_steps)"

Is it possible to set up something like this here? If so, how? And if not, what scheduler would u recommend to use with Prodigy? Or is it better to just stick with AdamW?
